app/services/decisions/income_decision_tree.rb

```diff
@@ -288,7 +288,7 @@ def edit_dependants(add_blank: false)
     def determine_showing_no_income_page
       if include_partner_in_means_assessment?
         edit(:partner_employment_status)
-      elsif crime_application.income&.all_income_over_zero?
+      elsif crime_application.income&.all_income_over_zero? || crime_application.income&.client_self_employed?
```

Comment:
We probably want to check that the user has an income from their self employment, not just that they are self employed. Not certain -  I'm checking what eForms does now

Reply:
Seems like they're not asked 👍